index.html

```diff
@@ -67,65 +81,88 @@ <h3 class="headthree">I am a Software Developer</h3>
     </ul>
     <br>
 </div>
-                                    <!--My works-->
+
+
+                                    <!--My works-->                         
 <section class="mainmyworks">
         <div class="worksheading">
-            <div>
+            <div class="workhead">
         <h2 class="headtwo">My Recent works</h2>
             </div>
             <div>
                 <hr class="liner">
             </div>
-        </div>
         <div class="workitems">
                     <div class="projectno1">
                         <div class="photo">
                         <img src="Images/Img Placeholder.png">
                         </div>
                     <div class="page">
                     <h4 class="headfour">Multi-Post Stories</h4>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                    </div>     
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>    
                 <ul class="pagelinks">
                         <li>css</li>
                         <li>html</li>
                         <li>bootstrap</li>
                         <li>Ruby</li>
-                </ul>
+                </ul> 
                 <button type="button" class="button">See Projects</button>
                     </div>
             </div>
+            </div>
+            </div>
+
+<div class="contents">
```

Comment:
It looks good, but please make sure to fix the sizing and spacing issues.

![image](https://user-images.githubusercontent.com/34779928/194695550-8c16a55c-1c65-4d63-90c8-0058cdc9ff4a.png)